scripts/vaccination_alerts/utils.ts

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+import {
+  RegionVaccinePhaseInfo,
+  stateVaccinationPhases,
+} from '../../src/cms-content/vaccines/phases';
+
+export interface RegionVaccinePhaseInfoMap {
+  [fipsCode: string]: RegionVaccinePhaseInfo;
+}
+
+export interface RegionVaccineVersion {
+  emailAlertVersion: number;
+}
+
+export interface RegionVaccineVersionMap {
+  [fipsCode: string]: RegionVaccineVersion;
+}
+
+export function getCmsVaccinationInfo(): RegionVaccinePhaseInfoMap {
+  return _.chain(stateVaccinationPhases)
```

Comment:
Is there a reason this is state-specific and doesn't include counties?

Reply:
We only have state information so far, but if we do get state information, we can import it here and export just a map with all the vaccination information